openwisp_monitoring/device/schema.py

```diff
@@ -145,6 +145,42 @@
                             "tx_carrier_errors": {"type": "integer"},
                         },
                     },
+                    "wireless": {
+                        "type": "object",
```

Comment:
Please add the following:

* additionalItems False
* additionalProperties false
* required: [channel, country... all of them]